project/src/main/java/com/google/sps/Models/Database.java

```diff
@@ -51,24 +51,26 @@ public static User logInUser(String email, String nickname) {
     Entity userEntity = getDatastore().prepare(query).asSingleEntity();
 
     if(userEntity != null) {
-      return new User(email, nickname, userEntity.getKey().getId());
+      ArrayList<String> docHashes = (ArrayList)userEntity.getProperty("docHashes");
```

Comment:
Are these changed in the document branch as well? If so, I think we should remove changes to Database, User, and Document to keep things separate.

Reply:
If we don't need the changes Database, etc. we can push straight to master